50-dracut.install

```diff
@@ -5,6 +5,12 @@ KERNEL_VERSION="$2"
 BOOT_DIR_ABS="$3"
 KERNEL_IMAGE="$4"
 
+# If KERNEL_INSTALL_MACHINE_ID is defined but empty, BOOT_DIR_ABS is a fake directory.
+# So, let's skip to create initrd.
+if ! [[ ${KERNEL_INSTALL_MACHINE_ID-x} ]]; then
```

Comment:
${FOO-x} equals to x if FOO is not defined, and equivalent to ${FOO} if defined. So the condition hits only if KERNEL_INSTALL_MACHINE_ID is defined but empty, as explained in the comment.